javascript/javascript1/week2/FlightBooking.js

```diff
@@ -0,0 +1,19 @@
+function getFullname(firstname,surname) {
+    return firstname,surname;
+}
+console.log("saloumeh","sarabi");
+
+let fullname1 = getFullname; 
+let fullname2 = getFullname;
+console.log("saloumeh sarabi,farzad soleimani");
+
```

Comment:
Same here. You are printing a string but you need to use the function instead :)